### PR TITLE
Adds Tokenize-rt recipe

### PR DESCRIPTION
**Pull request by sterlingbaldwin**
Creates recipe for tokenize-rt so it can be used as a depenency for future-fstrings.

### CODE REVIEW

File: recipes/tokenize-rt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install
```
Comment (mariusvniekerk) [inline recipes/tokenize-rt/meta.yaml]: can you switch this with `python -m pip install --no-deps --ignore-installed .`

File: recipes/tokenize-rt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (mariusvniekerk) [inline recipes/tokenize-rt/meta.yaml]: replace `setuptools` with `pip`

File: recipes/tokenize-rt/meta.yaml (hunk 1 above)
Comment (sterlingbaldwin) [reply recipes/tokenize-rt/meta.yaml]: No problem.

File: recipes/tokenize-rt/meta.yaml (hunk 2 above)
Comment (sterlingbaldwin) [reply recipes/tokenize-rt/meta.yaml]: Done

**Pull request merged**